src/System.Net.Http/src/System/Net/Http/Managed/HttpSystemProxy.cs

```diff
@@ -0,0 +1,165 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Net.Http;
+using System.Net;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+using SafeWinHttpHandle = Interop.WinHttp.SafeWinHttpHandle;
+
+namespace System.Net.Http
+{
+    internal sealed class HttpSystemProxy : IWebProxy
+    {
+        private readonly Uri _proxyUri;     // String URI for HTTPS requests
+        string[] _bypass = null;            // list of domains not to proxy
+        bool _bypassLocal = false;          // we should bypass domain considered local
+        private ICredentials _credentials=null;
+        private readonly WinInetProxyHelper _proxyHelper = null;
```

Comment:
- `_bypass` and `_bypassLocal` are missing `private`.
 - The above fields don't need to be explicitly set to `null` and `false`.